core/src/edu/asu/c3simulator/screens/BusinessCreationGuide.java

```diff
@@ -18,7 +19,7 @@ public BusinessCreationGuide(Game game)
 	{
 		resetAllGuideScreens();
 		//TODO add Industry screen
-		game.setScreen(new IndustryScreen(game));
+		game.setScreen(AllManagementScreens.INDUSTRY.getInstance());
```

Comment:
This looks like a method, not a class...
What does this class actually provide?